### ISSUE TITLE
Refused answer

### ISSUE THREAD

**Issue by pguth** [pguth/PDF.js_w-o_print_and_save#4]
I would have replied to @timvandermeij in the thread at mozilla/pdf.js/issues/2611 the following:

> Didn't read your comment complely. Flaw in the beginning of the argument, check https://github.com/pguth/PDF.js_w-o_print_and_save/commit/e8586f1fca93178cd9e196f8c892a8a0f25665cf and the commit time stamp to see that i was meaning it like i wrote it. 'Hope you accept that as proof.

**Comment by timvandermeij** [comment]
I really couldn't interpret your comment differently. If I have made a mistake with that, I am truly sorry. Negative comments about contributors were placed before and it makes me really sad, so I'm sorry if I overreacted.

I really appreciate your message here and I'm glad that there were no negative intentions with the comment. I have removed my comment and hopefully you will accept my apology!
